src/pages/home/report/ReportActionCompose/ReportActionCompose.js

```diff
@@ -338,7 +347,7 @@ function ReportActionCompose({
                                     reportID={reportID}
                                     report={report}
                                     reportParticipantIDs={reportParticipantIDs}
-                                    isFullComposerAvailable={isFullComposerAvailable}
+                                    isFullComposerAvailable={isFullComposerAvailable && !isCommentEmpty}
```

Comment:
This caused regression of not showing compose button on multiple new lines - https://github.com/Expensify/App/issues/28093#issuecomment-1737604866